### ISSUE TITLE
[dev] Colima mounts container/service volumes in read-only mode

### ISSUE THREAD

**Issue by lobsterkatie** [getsentry/seer#610]
When running docker through colima (as is the standard sentry practice) all of the volumes (folders that docker mirrors from the local repo into the container) are read-only, meaning you can't, for example, create a migration.

- Things I/we (Zach and I) tried in order to fix this:
  - Adding `:rw` to the end of the volume descriptions in `docker-compose.yml`. This shouldn't have made a difference, because it's theoretically the default, but I thought it was worth a try. No luck.
  - Changing permissions on the folder after it's copied over in the `Dockerfile`. Apparently this gets overridden by whatever you have locally when the volume mounts, so no on that one, too.
  - Changing permissions locally for the folders mirrored as volumes (`src`, `models`, and `tests`) so that everyone had write permissions, by running `chmod -R a+w src` to recursively and for all actors add write permissions. The new permissions were indeed reflected inside the container, but to no avail.
  - Eventually we realized that the one things Zach and Tillman (who didn't have this problem) had in common compared to Jodi and me (who did) is that they never got on the colima train. Sure enough, when I did `colima stop`, opened up Docker Desktop to get it start a docker process of its own, and then did `make shell` (after waiting for the image to rebuild; I guess colima and docker each keep their own copy, which I guess isn't surprising), I was able to `touch src/hi` without getting the read-only filesystem error.

TL;DR: Something about how colima implements volume mounting means that trying to write to a volume in a container (like we do when making migrations, for example) results in a `File system is read-only` error. Running docker through Docker Desktop does work, though, so for now that's the workaround.

TODO: Figure out how to get colima to play nicely with docker volumes.
